
{'date': 'Thu Jul 04 2019 11:33:10.947 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 131
    a_floresta = FlorestaBanana()
  module <module> line 124
    caverna = Caverna(self.floresta_inicio)
NameError: name 'Caverna' is not defined
'''},
{'date': 'Thu Jul 04 2019 11:40:33.336 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 120
  floresta_faca = CenaProxy(self.floresta_inicio)
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Oct 03 2019 11:03:33.580 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 70
    a_floresta = Caverna()
  module <module> line 58
    self.entrada_da_caverna = Elemento(A_ENTRADA, tit="a entrada da caverna escura", style=dict(
NameError: name 'A_ENTRADA' is not defined
'''},
{'date': 'Thu Oct 03 2019 11:18:18.716 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    from _spy.vitollino.main import Cena, Texto, INVENTARIO, STYLE
  module _spy.vitollino.main line 484
    class Elemento(Element_):
  module _spy.vitollino.main line 487
    def __init__(self, img="", vai=None, style=NDCT, tit="", alt="",
NameError: name 'NDCT' is not defined
'''},
{'date': 'Thu Oct 03 2019 11:31:36.739 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    from _spy.vitollino.main import Cena, Texto, INVENTARIO, STYLE
  module _spy.vitollino.main line 486
    class Elemento(Element_):
NameError: name 'Element_' is not defined
'''},
{'date': 'Thu Oct 03 2019 11:40:06.962 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 122
  self.entrada = Elemento(ENTRADA, tit=legenda, x=320, y=80, w=600, h=400, style={"opacity": 0.1} cena=caverna)
                                                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Oct 03 2019 12:06:10.822 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 150
    a_floresta = Caverna()
  module <module> line 131
    l.do_drag = True
UnboundLocalError: local variable 'l' referenced before assignment
'''},
{'date': 'Thu Oct 03 2019 12:44:46.537 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 83
    a_floresta = Caverna()
  module <module> line 70
    INVENTARIO.bota("lanterna","https://i.imgur.com/Z4DAh02.png",drag=True)
  module _spy.vitollino.main line 358
    item_elt = Elemrnto(item, tit=nome_item, w=30, height=30, drag=drag, style=ESTYLE)
NameError: name 'Elemrnto' is not defined
'''},